packages/fxa-auth-server/test/remote/subscriptions_tests.js

```diff
@@ -0,0 +1,50 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const { assert } = require('chai');
+const TestServer = require('../test_server');
+const Client = require('../client')();
+const config = require('../../config').getProperties();
+const testUtils = require('../lib/util');
+const oauthServerModule = require('../../fxa-oauth-server/lib/server');
+
+describe('/subscriptions/ routes', function () {
+  this.timeout(15000);
+  let client;
+  let email;
+  let oauthServer;
+  let password;
+  let server;
+
+  before(async () => {
+    testUtils.disableLogs();
+    oauthServer = await oauthServerModule.create();
+    await oauthServer.start();
+    server = await TestServer.start(config, false, {oauthServer});
+  });
+
+  after(async () => {
+    await TestServer.stop(server);
+    await oauthServer.stop();
+    testUtils.restoreStdoutWrite();
+  });
+
+  beforeEach(async () => {
+    email = server.uniqueEmail();
+    password = 'test password';
+    client = await Client.createAndVerify(config.publicUrl, email, password, server.mailbox);
+  });
+
+  it('successfully grants tokens for subscriptions management', async () => {
+    const res = await client.api.grantSubscriptionsManagementToken(client.sessionToken);
```

Comment:
Really minimal test here. Doesn't assert anything about IP binding or the like. Suggestions more than welcome!

Reply:
Just going to remove this route entirely.